reference/features/security.md

```diff
@@ -0,0 +1,266 @@
+# Security
+
+MeiliSearch allows you to protect your instances by using API keys. API keys give you fine-grained control over which users can access which indexes, endpoints, and  routes.
+
+## Protecting a MeiliSearch instance
+
+To protect a MeiliSearch instance, you must supply it with an alphanumeric string representing your `master` key. This can be done with the [master key configuration option](/reference/features/configuration.md#master-key):
+
+:::: tabs
+::: tab CLI
+
+```sh
+./meilisearch --master-key="[your_master_key]"
+```
+
+:::
+
+::: tab Environment variable
+
+```sh
+MEILI_MASTER_KEY="[your_master_key]"
+```
+
+:::
+::::
+
+::: note
+Launching an instance with a `master` key is mandatory when `env` is set to `production`.
+:::
+
+Communicating with a protected instance's endpoints requires adding a `Authorization` header to your request.
+
+### Using the `master` key
+
+Whenever possible, you should avoid using your `master` key to access API endpoints other than [`/keys`](/reference/api/keys.md) and [use API Keys instead](#api-keys). When you launch a secured instance for the first time, MeiliSearch creates a two default API keys: [`Default Search API Key` and `Default Admin API Key`](#default-api-keys).
+
+The `master` key is the only key with access to the [`/keys` endpoint](/reference/api/keys.md). You can use this endpoint to [create, update, and delete API keys](#api-keys).
```

Comment:
```suggestion
The `master` key is the only key with access to the [`/keys` endpoint](/reference/api/keys.md). You can use this endpoint to [create, update, delete, list API keys](#api-keys).
```

Reply:
Will use your suggestion, just adding a final "and" there.

```suggestion
The `master` key is the only key with access to the [`/keys` endpoint](/reference/api/keys.md). You can use this endpoint to [create, update, list, and delete API keys](#api-keys).
```